server/service/oauth.go

```diff
@@ -41,7 +39,8 @@ func NewOAuth(clientID, clientSecret string) *OAuth {
 	}
 }
 
-type githubUser struct {
+// GithubUser represents the user response returned by the GitHub auth.
+type GithubUser struct {
```

Comment:
I think it does not need to be public, because handlers does not build it, just use it